zucchini/graders/junit_xml_grader.py

```diff
@@ -69,7 +70,7 @@ def part_from_config_dict(self, config_dict):
         return JUnitXMLTest.from_config_dict(config_dict)
 
     def grade(self, submission, path, parts):
-        gradle_cmd = self.gradle_cmd
+        gradle_cmd = copy.deepcopy(self.gradle_cmd)
```

Comment:
we could just say `gradle_cmd = self.gradle_cmd[:]` fwiw

Reply:
right we don't actually need a deep copy